assets/panel/debugger.properties

```diff
@@ -416,6 +416,18 @@ callStack.collapse=Collapse rows
 # message to show more of the frames.
 callStack.expand=Expand rows
 
+# LOCALIZATION NOTE (callStack.group.expandTooltip): The text that will appear
+# when hovering a collapsed Group of frames in the callStack panel. `frames` is
+# always plural since a group can only exist if it contain more that 1 frame.
+# %S is replaced by the name of the library of the frames in the group.
+callStack.group.expandTooltip=Show %S frames
```

Comment:
Unfortunately "always plural" (i.e. more than 1) doesn't really matter, we need a proper plural form here.
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_and_Plurals

There should be already [usages in the code](https://github.com/devtools-html/debugger.html/blob/4bb14abb94fbb4361710600ac5e1c9c2da75c3f6/assets/panel/debugger.properties#L758-L765).

Reply:
I'm not sure what should be done here. What I meant by "always plural" is that it will always be: `Show React frames` (for example).
So there's no number I can pass to it to decide between singular and plural. Should I still be using a proper PluralForm if I don't have any numbers?

Reply:
Sorry, I completely missed that was the result before seeing the example, and was thrown off by the "always plural". You're correct: without a number, you can't have a correct plural form, so this should be interpreted as a generic "frames".

In the end, this is correct. I would only add the the "Show React frames" as an example in the comment for clarity.

Reply:
will do, thanks :)